CK3AGOT/localization/replace/german/agot/agot_rp_event_l_german.yml

```diff
@@ -27,37 +27,37 @@
  agot_rp_event.0001.c.tt:0 "Mit dieser Option wird das Ereignis 'Eine Dynastie entsteht' abgelehnt und Ihr startet ohne Modifikatoren oder Meinungsmodifikatoren für Euren Lehnsherren."
 
  agot_rp_event.0002.t:0 "Eine Dynastie entsteht"
- agot_rp_event.0002.desc:0 "'Dies wird nicht lange dauern; es geht um eine Zusammenfassung der Entstehung Eures Hauses, nicht um eine ausführliche Geschichte oder Ambitionen. Sollen wir damit beginnen, wie Ihr Eure Lordschaft erlangt habt und mit Eurer Vergangenheit?'"
+ agot_rp_event.0002.desc:0 "„Dies wird nicht lange dauern, es geht um eine Zusammenfassung der Entstehung Eures Hauses, nicht um eine ausführliche Geschichte oder Ambitionen. Sollen wir damit beginnen, wie Ihr Euren Titel erlangt habt und mit Eurer Vergangenheit?“"
  agot_rp_event.0002.a:0 "Ich war ein Jäger, der meinem Herrn in einer Zeit der Not geholfen hat."
  agot_rp_event.0002.a.tt:0 "Indem Ihr Eurem Lehnsherrn das Leben vor einem wilden Tier gerettet habt oder schlicht zu Hilfe gekommen seid, als er im Wald verletzt wurde, habt Ihr Euch Eure Lordschaft verdient."
- agot_rp_event.0002.b:0 "Ich war ein Krieger; meine Taten auf dem Schlachtfeld haben mir diesen Titel eingebracht."
- agot_rp_event.0002.b.tt:0 "Das Schlachtfeld hat Euch Ruhm und die Aufmerksamkeit Eures Herrn eingebracht, sei es durch die Hilfe, die Ihr ihm geleistet habt, oder einfach nur durch den Ruhm, den Ihr errungen habt; der Titel des Lords wurde Euch verliehen."
+ agot_rp_event.0002.b:0 "Ich war ein Krieger, meine Taten im Feld brachten mir diesen Titel ein."
```

Comment:
Auf dem Schlachtfeld finde ich tatsächlich besser in dem Kontext, wenn dann "zu Felde"